nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+app.get("/numbers/add", (req, res) => {
+    const firstNum = parseInt(req.query.first);
```

Comment:
The code would not work correctly if the parameters are not numbers.
`GET /numbers/add?first=abc&second=ggdsdfs`